src/firefox/remote.js

```diff
@@ -16,11 +16,13 @@ export const REMOTE_PORT = 6005;
 
 
 // RemoteFirefox types and implementation
-
+/* eslint-disable import/order, import/imports-first, import/no-extraneous-dependencies */
+//This is related to Flow and will be stripped in production hence won't be treated as a dependency at all
```

Comment:
when we have regular comments in the same place where we are going to use an eslint-disable block comment, it would be probably better to move that comment outside the eslint-disable, e.g.:

```
// This is related to ...
/* eslint-disable ... */
...
/* eslint-enable ... */
```

and 

```
// import Flow Types
/* eslint-disable ... */
...
/* eslint-enable ... */